src/core/db/models.py

```diff
@@ -60,6 +60,22 @@ def __repr__(self):
         return f"<User {self.telegram_id}>"
 
 
+class ExternalSiteUser(Base):
+    """Модель пользователя с сайта ProCharity."""
+
+    __tablename__ = "external_site_users"
+
+    id_hash: Mapped[str] = mapped_column(String(256), nullable=False)
+    email: Mapped[str] = mapped_column(String(48), unique=True, nullable=False)
+    first_name: Mapped[str] = mapped_column(String(64), nullable=True)
+    last_name: Mapped[str] = mapped_column(String(64), nullable=True)
+    specializations: Mapped[str] = mapped_column(String(), nullable=False)
+    source: Mapped[str] = mapped_column(String(), nullable=True)
```

Comment:
Тут нет смысла указывать тип, он и так автоматом ставится. 
```suggestion
    source: Mapped[str] = mapped_column(nullable=True)
```